_i18n/es/resources/user-guides/create_wallet.md

```diff
@@ -1,62 +1,60 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-### Operating Systems:  Ubuntu
+{% include disclaimer.html translated="true" version=page.version %}
+### Sistemas Operativos:  Ubuntu
 
-- Download the [official binaries](https://getmonero.org/downloads/) or compile the last source available on [Github](https://github.com/monero-project/bitmonero)
+- Descarga los [binarios oficiales](https://getmonero.org/downloads/) o compila la última fuente disponible en [Github](https://github.com/monero-project/bitmonero)
 
 ![image1](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/1.png)
 ![image2](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/2.png)
 
-- Extract the files with the archive manager (same as Winzip on Windows). Note the path where the files "monerod" and "monero-wallet-cli" are
+- Extrae los archivos con el administrador de archivos (igual que Winzip en Windows). Anota dónde se encuentran los archivos "monerod" y "monero-wallet-cli"
 
 ![image3](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/3.png)
 ![image4](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/4.png)
 
-- You only need to do this step once : open a terminal (ctrl+alt+t) and install the required dependencies by typing : "*sudo apt-get install libboost-all-dev libssl-dev libevent-dev libdb++-dev*". When asked, press the Y key and then Enter to continue
+- Sólo necesitas hacer este paso una vez: abre una terminal (ctrl+alt+t) e instala las dependencias requeridas escribiendo: "*sudo apt-get install libboost-all-dev libssl-dev libevent-dev libdb++-dev*". Cuando se pregunte, presiona la tecla Y y después Enter para continuar.
 
 ![image5](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/5.png)
 ![image6](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/6.png)
 
-- Open a terminal and load the path where your binaries are extracted (cf. step 2) by typing : "*cd yourPathFromStep2*"
+- Abre una terminal e ingresa la dirección donde fueron extraídos tus binarios (paso 2) escribiendo: "*cd yourPathFromStep2*"
 
 ![image7](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/create_wallet/7.png)
 
-- Load monerod by typing in your terminal : "*./monerod*". Wait for the synchronization with the network (monerod is updating the blockchain you have downloaded in step 4 or is downloading it from scratch). This can take a lot of time the first time, so be patient
+- Carga monerod escribiendo en la terminal: "*./monerod*". Espera por la sincronización con la red (monerod está actualizando la blockchain que has descargado en el paso 4 o la está descargando desde cero).
```

Comment:
Te hizo falta la última parte.

 "This can take a lot of time the first time, so be patient".

Reply:
Debí haberlo borrado antes de traducir y olvidado traducirlo, gracias.